stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -10,19 +10,97 @@
 package com.facebook.stetho.inspector.elements.android;
 
 import android.view.View;
+import android.view.ViewDebug;
+import com.facebook.stetho.common.LogUtil;
 import com.facebook.stetho.common.StringUtil;
 import com.facebook.stetho.common.android.ResourcesUtil;
-import com.facebook.stetho.inspector.elements.AttributeAccumulator;
 import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.AttributeAccumulator;
+import com.facebook.stetho.inspector.elements.StyleAccumulator;
+import com.facebook.stetho.inspector.helper.IntegerFormatter;
 
 import javax.annotation.Nullable;
+import javax.annotation.concurrent.GuardedBy;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
+import java.util.regex.Pattern;
 
 final class ViewDescriptor extends AbstractChainedDescriptor<View> implements HighlightableDescriptor {
-  private static final String ID_ATTRIBUTE_NAME = "id";
+  private static final String ID_NAME = "id";
+  private static final String NONE_VALUE = "(none)";
+  private static final String NONE_MAPPING = "<no mapping>";
 
   private final MethodInvoker mMethodInvoker;
 
+  /**
+   * NOTE: Only access this via {@link #getWordBoundaryPattern}.
+   */
+  @Nullable
+  private Pattern mWordBoundaryPattern;
+
+  /**
+   * NOTE: Only access this via {@link #getProperties}.
+   */
+  @Nullable
+  @GuardedBy("this")
+  private volatile Set<ViewCSSProperty> mViewProperties;
+
+  private Pattern getWordBoundaryPattern() {
+    if (mWordBoundaryPattern == null) {
+      mWordBoundaryPattern = Pattern.compile("(?<=\\p{Lower})(?=\\p{Upper})");
+    }
+
+    return mWordBoundaryPattern;
+  }
+
+  private Set<ViewCSSProperty> getProperties() {
```

Comment:
I'm a big fan of symmetry/consistency in the field name and method name ... so I recommend renaming this method to `getViewProperties` (alternately you could rename the field to `mProperties` I suppose)